db/migrate/20160309182642_remove_social_events_table.rb

```diff
@@ -0,0 +1,134 @@
+class RemoveSocialEventsTable < ActiveRecord::Migration
```

Comment:
Would that be sth like https://github.com/openSUSE/osem/pull/855/commits/f9bf080de292748489e1e3299a0810c961e58211 ?
Would we have to invoke the rake task from within the migration that drops the columns, so that we ensure that the rake task is executed **prior** to the destruction of data?